### PR TITLE
Add destroy action and button for action, tests, confirmation.

### PR DESCRIPTION
**Pull request by Krzyfho**
Add delete action, tests, button with bootstrap.

### CODE REVIEW

File: app/views/expenses/index.html.erb [hunk 1]
```diff
@@ -20,6 +20,8 @@
         <td><%= exp.paid_on %></td>
         <td>
           <%= link_to t('expenses.index.edit'), edit_expense_path(exp.id), class: "btn btn-outline-primary"%>
+          <%= link_to t('expenses.index.destroy'), expense_path(exp.id), method: :delete,
+                      data: { confirm: "Are you sure?" }, class: "btn btn-outline-danger"%>
```
Comment (hesler) [inline app/views/expenses/index.html.erb:24]: Translations?

File: spec/controllers/expenses_controller_spec.rb [hunk 2]
```diff
@@ -189,4 +189,35 @@
...
+
+      it 'create flash message' do
+        expect(flash[:success]).to eq I18n.t('flash.controller.good_delete')
+      end
+
+      it 'has 302:Redirect' do
+        expect(response).to have_http_status(:redirect)
+      end
+
+      it 'save record to database with success' do
```
Comment (hesler) [inline spec/controllers/expenses_controller_spec.rb:218]: It rather removes a record, doesn't it? :D

**Review by hesler — APPROVED** [review]

File: spec/controllers/expenses_controller_spec.rb (hunk 2 above)
Comment (Krzyfho) [reply spec/controllers/expenses_controller_spec.rb:218]: Yes, yes XD

File: app/views/expenses/index.html.erb (hunk 1 above)
Comment (Krzyfho) [reply app/views/expenses/index.html.erb:24]: I want to add modal from bootstrap for better view, but wherever i search i see only JavaScript script and HTML.

**Pull request merged**